core/constants_test.py

```diff
@@ -40,6 +40,15 @@ def test_constants_file_contains_valid_json(self) -> None:
             self.assertTrue(isinstance(json, dict))
             self.assertEqual(json['TESTING_CONSTANT'], 'test')
 
+    def test_loading_non_existing_file_throws_error(self) -> None:
```

Comment:
This is not typically how you would write a test I believe. Doesn't it work without a try except, where you do `with self.assertRaisesRegex(...): callFunction()`?

Reply:
Ah, my bad, I have changed implementation PTAL.
with the new implementation, this test is no longer needed I think.